src/pages/messages/TasksPage.tsx

```diff
@@ -574,7 +584,7 @@ const TasksPage = () => {
           <PrivateRoute exact path={UrlPathTemplate.TasksNvi} isAuthorized={isNviCurator}>
             <NviCandidatesList
               nviCandidatesQuery={nviCandidatesQuery}
-              nviListQuery={nviListQuery}
+              nviQueryParams={listNviCandidatesParams}
```

Comment:
Her ser det ut som du trenger params-objektet for å bruke det et annet sted. Ved å innkapsle objektet vil du kanskje ikke trenge å sende det inn når du bruker hooken som har objektet. Men skulle du trenge det kan det også returneres fra hooken som innkapsler det.